CHANGELOG.md

```
@@ -9,37 +9,29 @@
-
+- dependabot.yml ([be0056b](https://github.com/VilnaCRM-Org/php-service-template/commit/be0056bf3f16bb79826d51b6aadf42aa3d729437))
 
 # [1.6.0](https://github.com/VilnaCRM-Org/php-service-template/compare/v1.5.0...v1.6.0) (2025-05-05)
 
-
 ### Features
 
-* improve dependabot updates ([253b194](https://github.com/VilnaCRM-Org/php-service-template/commit/253b19476e38bb5be219488eb54f5d33f78da2b3))
-
-
+- improve dependabot updates ([253b194](https://github.com/VilnaCRM-Org/php-service-template/commit/253b19476e38bb5be219488eb54f5d33f78da2b3))
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)